### DISCUSSION TITLE
Please drop links to the best bitcoin technical documentation and cultural writing!

### DISCUSSION THREAD

**Discussion by casey** [ordinals/ord#1738] [Show and tell]

**Comment by Kodylow** [comment]
Grokking bitcoin by kalle https://rosenbaum.se/book/grokking-bitcoin.html

**Comment by andrewtoth** [comment]
https://www.lopp.net/bitcoin-information.html

**Comment by Blockchainwolf** [comment]
1. Bitcoin : https://developer.bitcoin.org/devguide/ 

2. BDN Developer Network :https://bitcoindev.network 

3. BIP : https://bips.dev 

4. Best Youtube For Technical : https://www.youtube.com/channel/UCywSzGiWWcUG1gTp45YdPUQ/videos 

5.Dev:  https://bitcoinbook.dev/#/

6. Bitcoin Simulator :  https://www.bitcoinsimulator.tk/blockchain?chain=public

7. All the tech part of bitcoin : https://bitcoinops.org 

8.Bitcoin Developer Reference
Working Paper : https://www.lopp.net/pdf/Bitcoin_Developer_Reference.pdf 

9.Bitcoin Wiki :  https://en.bitcoin.it/wiki/Main_Page 

10. Resoruces https://github.com/Xel/Blockchain-stuff#bitcoin
